co-log-benchmark/co-log-benchmark.cabal

```diff
@@ -23,10 +22,11 @@ executable co-log-features-bench
   main-is:             Main.hs
   -- other-modules:
   -- other-extensions:
-  build-depends:       base >=4.10 && <4.13,
+  build-depends:       base-noprelude >=4.10 && <4.13,
```

Comment:
Heh. `Prelude` in the `co-log` package is now in `other-modules` so `base-noprelude` trick won't work. Anyway, you didn't used `relude` before in `co-log-benchmarks` so I don't expect that addition of `relude` is required... Need to see CI failure to check what's wrong

Reply:
I need to either use Prelude trick:
```
import Data.Semigroup
import Prelude
```

so semigroup will be imported in all GHC versions, or use `relude`.